data/migrations/20200124081059_my-projects.js

```diff
@@ -0,0 +1,61 @@
+exports.up = function(knex) {
+  return knex.schema
+    .createTable("projects", table => {
+      table.increments("projectID");
+      table
+        .string("title", 128)
+        .notNullable()
+        .unique();
+      table.text("projectDescription");
+      table.boolean("completed").defaultTo(false);
+    })
+    .createTable("resources", table => {
+      table.increments("resourceID");
+      table
+        .string("resourceName", 140)
+        .notNullable()
+        .unique();
+      table.text("resourceDescription");
+    })
+    .createTable("projectResources", table => {
+      table.increments();
+      table
+        .integer("projectID")
+        .unsigned()
+        .notNullable()
+        .references("projectID")
```

Comment:
Not a rule, but maintaining a given case ensures consistency, e.g `projectID` should be `projectId`